client/blocks/sharing-preview-pane/index.jsx

```diff
@@ -124,7 +124,13 @@ class SharingPreviewPane extends PureComponent {
 					/>
 				);
 			case 'tumblr':
-				return <TumblrSharePreview { ...previewProps } />;
+				return (
+					<TumblrSharePreview
+						{ ...previewProps }
+						articleContent={ post.content }
```

Comment:
`post.content` is not truncated, which is what we want here: truncation is handled in the `social-previews` package.